**Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. Найдите вероятность того, что стрелок попадет в цель ровно 85 раз.**

Воспользуемся формулой **Бернулли**:

$P_{n}(X=k)=C^k_{n}p^{k}q^{n-k}$, где 

                                        n=100 - количество испытаний,   
                                        k=85 - количество попадания в цель,  
                                        p=0.8 - вероятность попадания в цель,  
                                        q=0.2 - вероятность промаха

Создадим функцию расчета по формуле Бернулли:

In [1]:
from math import factorial, exp
def bernulli(n, k, p):
    comb=factorial(n)/(factorial(k)*factorial(n-k))
    return comb*(p**k)*(1-p)**(n-k)

и вычислим результат:

In [2]:

print(f'P = {bernulli(100,85,0.8): .4f}')

P =  0.0481


___  
_Ответ: вероятность того, что стрелок попадет в цель ровно 85 раз составляет **P=0.0481** или **$\approx$ 4.81%**_

___